code/weapon/weapon.h

```diff
@@ -251,6 +252,8 @@ typedef struct beam_weapon_info {
 	float range;						// how far it will shoot-Bobboau
 	float damage_threshold;				// point at wich damage will start being atenuated from 0.0 to 1.0
 	float beam_width;					// width of the beam (for certain collision checks)
+	bool beam_light_flicker;			// If beam light is affected by the flickering
+	bool beam_light_as_multiplier;		// If beam light is used directly or multiplies the section width
```

Comment:
Why not add these to the flag set?

Reply:
mainly I forgot flagsets exist. Will correct.

Reply:
Upon review and discussion, adding these(and the other bool later) to the existing regular flag sets would have undesirable side effects. To my mind creating new flagsets for them would be added complexity for little to no benefit, so I'm inclined to leave them as is unless there's a compelling benefit I'm unaware of.
The other review items have been addressed, though.

Reply:
What undesirable side effects would there be to adding it to beam info flags?

Reply:
It was my understanding that if anyone adds a beam flag in a tbm it will clear any beam flags set upstream, which isn't something I much care for really. Even if I was misinformed or misunderstood and that doesn't apply here, I'd still be curious to know what benefit that system provides over this method.

Reply:
Is this resolved or awaiting followup?

Reply:
@TRBlount I took it as resolved, personally, but I could be wrong

Reply:
It's resolved.  The only existing flagset is the weapon flags, which is not appropriate for this.